src/main/java/it/gov/pagopa/payhub/activities/service/paymentsreporting/FlussoRiversamentoHandler.java

```diff
@@ -0,0 +1,57 @@
+package it.gov.pagopa.payhub.activities.service.paymentsreporting;
+
+import it.gov.digitpa.schemas._2011.pagamenti.CtFlussoRiversamento;
+import it.gov.pagopa.payhub.activities.exception.ActivitiesException;
+import it.gov.pagopa.payhub.activities.service.XMLUnmarshallerService;
+import jakarta.xml.bind.JAXBContext;
+import jakarta.xml.bind.JAXBException;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.core.io.Resource;
+import org.springframework.stereotype.Component;
+import org.xml.sax.SAXException;
+
+import javax.xml.XMLConstants;
+import javax.xml.validation.Schema;
+import javax.xml.validation.SchemaFactory;
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Handles the deserialization of files conforming to the "FlussoRiversamento" schema.
+ */
+@Lazy
+@Component
+public class FlussoRiversamentoHandler {
```

Comment:
This class is not handling anything, it's just unmarshalling
```suggestion
public class FlussoRiversamentoUnmarshallerService {
```